src/less/components/loader.less

```diff
@@ -14,19 +15,13 @@
     }
 
     &.loader-sm {
-        width: 2rem;
-        height: $width;
+        width: 1.5rem;
+        height: 1.5rem;
     }
 
-    &.loader-lg {
-        border-width: 0.25rem;
-        width: 3.5rem;
-        height: $width;
-    }
-
-    &.loader-muted {
-        border-color: @light-gray;
-        border-left-color: @text-gray;
+    &.inverted {
+        border-color: #ffffff66;
+        border-left-color: @white;
```

Comment:
We have other utility classes prefixed with `loader-` should this one also be prefixed?